roles/nos_social/templates/docker-compose.yml.tpl

```diff
@@ -22,6 +22,8 @@ services:
       - "traefik.enable=true"
       - "traefik.http.routers.nip05api.rule=(Host(`{{ domain }}`) && (PathPrefix(`/metrics`) || PathPrefix(`/api/`) || PathPrefix(`/.well-known`))) && !HostRegexp(`{subdomain:[a-zA-Z0-9-]+}.{{ domain }}`)"
       - "traefik.http.routers.nip05api.entrypoints=websecure"
+      - "traefik.http.middlewares.nip05api.ratelimit.average={{ nip05api_ratelimit_average }}"
```

Comment:
It's great how traefik lets you declaratively delegate some common functionality to the infra layer, feels very clean